cypress/support/commands.js

```diff
@@ -43,13 +43,57 @@ Cypress.Commands.add("openListsTab", () => {
 })
 
 Cypress.Commands.add("searchKeyword", (keyword) => {
+    cy.visit("/biotestmine/search");
     cy.get(".searchform > input").type(keyword + '{enter}',{delay:100});
 })
 
 Cypress.Commands.add("selectFromDropdown", (keyword) => {
     cy.get('select').then($option => {$option.val(keyword)}).parent().trigger('change');
 })
 
+Cypress.Commands.add("createGeneList", (geneList) => {
+    cy.contains("Upload").click();
+    cy.get(".wizard").find("textarea").type(geneList,{delay:100});
+    cy.contains("Continue").click();
+    cy.contains("Save List").click();
+    cy.wait(100);
```

Comment:
When I look at the CI test recording, it seems this wait isn't always enough. What will happen is that it will click the *Save List* button, then click *Lists* and Bluegenes will change to the lists page first and then to the query results page (as it waits for the list to be saved successfully before showing it, which makes it occur after changing to the list page).

Can you try replacing the `wait` with `cy.url().should("include","/results");`? The assertion should force cy.url() to retry until it passes, meaning Cypress won't continue until the page has actually changed.